.gitlab-ci.yml

```diff
@@ -230,11 +220,7 @@ before_script:
       - $ONLY_WINDOWS == "true"
   interruptible: true
   script:
-    - set -e
```

Comment:
Did you mean to get rid of these?

Reply:
Yes
They're useless in general too but especially in this case where there's just 1 command.